stl/inc/format

```diff
@@ -586,7 +586,8 @@ constexpr void _Parse_format_string(basic_string_view<_CharT> _Format_str, _Hand
                 // In this case we didn't find either a closing curl or opening curl.

```

Comment:
I think this comment is misleading. We know there are no open or close curls in `[_Begin, _OpeningCurl)`, but we may have found an open curl on 582. 
```suggestion
                // In this case there are neither closing nor opening curls in [_Begin, _OpenCurl)
```